### PR TITLE
fix blank shipping data cancel order

### LINKED ISSUE #3075: Shipping data area is blank on Cancelled orders
### Expected behavior
Shipping area should do one of the following on a cancelled order (or really any order), if there is no data in the shipping area.

1. Be removed
1. Show a "No shipping info" message

### Actual Behavior
There is a blank box where the shipping data should be

### Steps to Reproduce the Behavior
As an admin, cancel an order before doing any shipping status updates.

### Versions
```
Node: 6.11.3
NPM: 5.5.1
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.13.2
Reaction: 1.5.0
Reaction branch: release-1.5.0
Docker: 17.09.0-ce
```

reaction

### PR DESCRIPTION
**Pull request by iakhator**
Resolve #3075 

This PR try to display a message when an order is cancelled or removed. A blank area is shown in the shipping section whenever an order is cancelled. 

#### Test
* Loggin as Admin.
* Place an order.
* Checkout  and complete order.
* Click on order icon at the right.
* On the order row, click on the blue box with a white arrow.
* The order summary will slide out.
* Scroll to the `invoice` section.
* Toggle the `Approve` button to reveal `Cancel`.
* Click on the `Cancel` button.
* Scroll down to shipping section to see the `shipping info` displayed.

### CODE REVIEW

**Comment by iakhator** [comment]
cc @kieckhafer

File: imports/plugins/core/orders/client/templates/workflow/shippingTracking.html [hunk 1]
```diff
@@ -67,6 +67,8 @@ <h3 class="panel-title" data-i18n="orderWorkflow.{{toCamelCase label}}"><strong>
           {{/if}}
         </div>
         {{/if}}
+      {{else}}
+        <span>No shipping information available</span>
```
Comment (zenweasel) [inline imports/plugins/core/orders/client/templates/workflow/shippingTracking.html]: This needs a translation

Comment (iakhator) [reply imports/plugins/core/orders/client/templates/workflow/shippingTracking.html]: alright. I fix that

**Comment by spencern** [comment]
eslint passes locally after merging master, so I think this is a linter issue from a few releases ago.

**Pull request merged**